### DISCUSSION TITLE
I cannot reactivate the teacher program.

### DISCUSSION THREAD

**Discussion by ioisup** [community/community#118264] [GitHub Education]
### Hi everyone,

I upload my ID card with name, title, and expire date in the front and my school name in the back but it said cannot found my school name. May be because of the ID card is written in Thai.

**Comment by samus-aran** [comment]
Hi @ioisup 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I need to translate my evidence. What can I do?](https://github.com/orgs/community/discussions/111352#translate-my-evidence)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
